recipes-debian/corosync/corosync/corosync

```diff
@@ -0,0 +1,116 @@
+#! /bin/sh
```

Comment:
This file is not installed on sysv init environment.

```
root@qemuarm64:~# ls /etc/init.d/
banner.sh                 dbus-1                    hostname.sh               ofono                     reboot                    stop-bootlogd             urandom
bluetooth                 devpts.sh                 hwclock.sh                pacemaker                 rmnologin.sh              sysfs.sh
bootlogd                  dmesg.sh                  modutils.sh               populate-volatile.sh      run-postinsts             syslog
bootmisc.sh               dropbear                  mountall.sh               rc                        save-rtc.sh               systemd-udevd
checkroot.sh              functions                 mountnfs.sh               rcS                       sendsigs                  umountfs
connman                   halt                      networking                read-only-rootfs-hook.sh  single                    umountnfs.sh
```

Reply:
I am sorry. I fixed it.
```
root@qemuarm64:/etc/init.d# ./corosync start
Starting corosync daemon: corosyncMar 03 04:18:31 notice  [MAIN  ] Corosync Cluster Engine 3.0.1 starting up
Mar 03 04:18:31 info    [MAIN  ] Corosync built-in features: dbus systemd xmlconf snmp relro bindnow
.
root@qemuarm64:/etc/init.d# ./corosync status
corosync is running.
root@qemuarm64:/etc/init.d# ./corosync stop
Stopping corosync daemon: corosync.
root@qemuarm64:/etc/init.d# ./pacemaker start
Starting Pacemaker Cluster Manager[  OK  ]
root@qemuarm64:/etc/init.d# ./pacemaker status
pacemakerd (pid 2149) is running...
root@qemuarm64:/etc/init.d# ./pacemaker stop
Signaling Pacemaker Cluster Manager to terminate[  OK  ]
Waiting for cluster services to unload[  OK  ]
```

Reply:
thanks.